registry/repository_test.go

```diff
@@ -287,12 +302,34 @@ func (c mockRemoteClient) Do(req *http.Request) (*http.Response, error) {
 		default:
 			return &http.Response{}, fmt.Errorf(msg)
 		}
+	case "/v2/test/referrers/sha256:0000000000000000000000000000000000000000000000000000000000000000":
+		return &http.Response{
+			StatusCode: http.StatusOK,
+			Body:       io.NopCloser(bytes.NewReader([]byte(validPageImage))),
+			Header: map[string][]string{
+				"Oras-Api-Version": {"oras/1.1"},
+			},
```

Comment:
We don't support ORAS artifact spec any longer. Should we remove those ORAS artifact spec related code?